recurly/__init__.py

```diff
@@ -423,6 +423,32 @@ class BillingInfo(Resource):
     sensitive_attributes = ('number', 'verification_value', 'account_number', 'iban')
     xml_attribute_attributes = ('type',)
 
+    def verify(self, gateway_code=None):
```

Comment:
Feel like it would be better to use `**kwargs` here but I had a hard time configuring this method to access the `Verify` class's attributes. Instead I'm appending elements like `elem.append(Resource.element_for_value(Verify.attributes[0], gateway_code))` and it seems brittle.